services/auth/signin.go

```diff
@@ -27,7 +27,7 @@ func UserSignIn(username, password string) (*models.User, *login.Source, error)
 	if strings.Contains(username, "@") {
 		user = &models.User{Email: strings.ToLower(strings.TrimSpace(username))}
 		// check same email
-		cnt, err := db.Count(user)
+		cnt, err := db.GetEngine(db.DefaultContext).Count(user)
```

Comment:
Realistically there should be a CountUser function in models that means that we should not be exposing the db to services.

Reply:
The code was there. The code was accessing database directly via db.Xxx.

The new code is just more clear than before.

Adding new packages is not the task or purpose of this PR. This PR just makes a clean up.